src/main/java/org/junitpioneer/jupiter/params/ByteOrder.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016-2022 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.params;
+
+/**
+ * Utility enum because {@link java.nio.ByteOrder} is not an enum.
+ */
+public enum ByteOrder {
```

Comment:
What about moving this enum into the annotation definition? In the style of [`@EnumSource`](https://github.com/junit-team/junit5/blob/r5.9.3/junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/EnumSource.java) and its [`Mode`](https://github.com/junit-team/junit5/blob/bd7d03f517dfca3730fbd123c5fd4d4b70930129/junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/EnumSource.java#L95-L181) enum.